src/libmongoc/doc/mongoc_client_encryption_encrypt_text_opts_destroy.rst

```diff
@@ -0,0 +1,26 @@
+:man_page: mongoc_client_encryption_encrypt_text_opts_destroy
+
+mongoc_client_encryption_encrypt_text_opts_destroy()
+==================================
```

Comment:
Fix underlines in docs files to fix the `WARNING: Title underline too short.` warnings causing failures in [make-docs](https://spruce.mongodb.com/task/mongo_c_driver_smoke_make_docs_patch_a7c78f67e32b7ee16b1f317f5af80452dc3701ab_690254883f88aa0007f76ae7_25_10_29_17_53_14/logs?execution=0).